### ISSUE TITLE
Apps not available via HTTPS

### ISSUE THREAD

**Issue by acosta11** [cloudfoundry/cf-for-k8s#46]
## Summary
HTTPS traffic does not reach apps running on cf-for-k8s.

## Repro Steps
1. Deploy cf-for-k8s and log in via the cf CLI
1. Enable feature flag 'diego_docker'
1. Target a space and push a docker app
1. Attempt to curl the route provided in the push output and observe it is only available over unencrypted http.
```
> cf push myapp -o cloudfoundry/diego-docker-app

> curl -i http://myapp.<app-domain>
HTTP/1.1 200 OK
date: Wed, 26 Feb 2020 00:21:36 GMT
content-length: 1
content-type: text/plain; charset=utf-8
x-envoy-upstream-service-time: 3
server: istio-envoy

0

> curl -i https://myapp.<app-domain> -k
HTTP/2 404
date: Wed, 26 Feb 2020 00:21:18 GMT
server: istio-envoy
```

## Context
Currently, there are two istio ingress gateways: one for the cf-system components and one for cf-workloads. The ingress gateway for cf-workloads namespace only configures a rule for unencrypted http traffic to port 80. Meanwhile the system component ingress gateway only listens for encrypted https traffic on port 443. So any encrypted https traffic to port 443 only matches the routing rule on the cf-system gateway and receives an istio 404 response. We expect to work with the cf-for-k8s networking team to resolve this issue.

**Comment by loewenstein** [comment]
Has this been solved by #64?

**Comment by XanderStrike** [comment]
#64 provided documentation for how users could enable TLS by providing their own certs. The story title was misleading, I've updated the PR to better reflect the work that was actually done.

The story to enable TLS for ingress by default is [here](https://www.pivotaltracker.com/n/projects/2407973/stories/171870086) and it is prioritized high in our backlog.

**Comment by jamespollard8** [comment]
Closed by #179
